pymc/distributions/continuous.py

```diff
@@ -1943,6 +1948,12 @@ def dist(
 
         return super().dist([alpha, m], **kwargs)
 
+    def get_moment(rv, size, alpha, m):
+        mean = alpha * m / (alpha - 1)
```

Comment:
Hmm this moment isn't great when alpha <= 1. It was also not great in V3!

Perhaps we should use the median or switch between the mean and the median when alpha <= 1?

Reply:
Thanks for this! I had that question on my mind as well! I think the switch might be confusing when using it. I'd much rather do the switching to median sounds better. What do you think?

Reply:
Which one do you mean. The two option I see are:
1. Use median regardless of alpha
2. Switch between mean and median based on alpha

I think I like option 1 better but perhaps there's a good argument for 2 (or 3 if you have any)

Reply:
I also meant that it should be option 1. I'll change the implementation as such. Thanks!